aip/general/0003.md

```diff
@@ -0,0 +1,34 @@
+---
+id: 3
+state: approved
+created: 2023-03-28
+placement:
+  category: meta
+  order: 30
+---
+
+# AIP Versioning
+
+This AIP defines the versioning scheme of the AIPs.
+
+## Guidance
+
+- The AIPs **must** be versioned by date, using the [ISO-8601 format of `YYYY-MM-DD`](https://www.w3.org/TR/NOTE-datetime), that corresponds to the date the version was added.
+- AIPs versions **must** be available as a tag on the source control mechanism used to store the AIPs, of the format `v{version}`. Example: `v2023-03-28`.
+- The AIPs **should** have a new version every 3 months, or when a significant change to an AIP occurs.
+
+## Appendix
+
+As the AIPs continue to evolve, it is beneficial for
+the AIPs to have a way to denote older versions of the
+specification to serve as a reference point for APIs
+to explain the rationale of their design choices.
+
+This becomes especially critical as AIPs can sometimes
+reverse guidance, or include an updated design pattern
+that may not look similar to the original pattern.
+
+The rational for date-based versioning is:
+
+- It allows for a simple heuristic to correlate the version of AIPs an API followed (look at the date of the API change, then look at the AIP version published right before that date).
```

Comment:
> It provides a simple means to order changes to AIPs. Since the changes are listed in the change log at the bottom of each AIP, simply compare the dates their with version date to see the previous version.

Reply:
Your rewrite is saying something a bit different from what I'm saying: in this proposal, the individual AIPs do not have a version. The total set of AIPs do.

So my statement is that one could pull up the version of this repository with the particular stamp to see the full AIP guidance at that time. 

Viewing the changelog is also valuable, so I definitely will add that note in the rationale section!

Reply:
Ok, edit as you see fit, however,

1. This is a runon sentence: sentences need to follow grammatical rules with or without parentheticals.
2. "to correlate the version of AIPs an API follows" is cumbersome and difficult to understand. Please fix.

Reply:
ack. agreed this longer than it needs to be. I'm opting to be more terse and hoping people can read between the lines a little bit. PTAL.